### PR TITLE
fix dolibarr version eval

### PR DESCRIPTION
**Pull request by cfoellmann**
for us on dolibarr 18.0.3 + php 8.2 the action on leftmenu is broken.

This is because the Dolibarr version can not be compared to the requirement in the if clause.

### CODE REVIEW

File: class/actions_searcheverywhere.class.php [hunk 1]
```diff
@@ -10,7 +10,7 @@ class ActionsSearcheverywhere
 	function printSearchForm($parameters, &$object, &$action, $hookmanager) {
 		global $langs,$db,$conf;
 
-		if (in_array('searchform',explode(':',$parameters['context'])) && (DOL_VERSION <= 3.8
+		if (in_array('searchform',explode(':',$parameters['context'])) && (floatval(DOL_VERSION) <= floatval('3.8')
```
Comment (atm-john) [inline class/actions_searcheverywhere.class.php:13]:
```suggestion
		if (in_array('searchform',explode(':',$parameters['context'])) && (floatval(DOL_VERSION) <= 3.8)
```

3.8 is already a float

File: class/actions_searcheverywhere.class.php [hunk 2]
```diff
@@ -101,7 +101,7 @@ function printSearchForm($parameters, &$object, &$action, $hookmanager) {
 	function addSearchEntry($parameters, &$object, &$action, $hookmanager) {
 		global $langs, $db, $conf;
 
-		if (in_array('searchform',explode(':',$parameters['context'])) && DOL_VERSION > 3.8 && empty($conf->global->SEARCHEVERYWHERE_SEARCH_PREVIEW)) {
+		if (in_array('searchform',explode(':',$parameters['context'])) && floatval(DOL_VERSION) > floatval('3.8') && empty($conf->global->SEARCHEVERYWHERE_SEARCH_PREVIEW)) {
```
Comment (atm-john) [inline class/actions_searcheverywhere.class.php:104]:
```suggestion
		if (in_array('searchform',explode(':',$parameters['context'])) && floatval(DOL_VERSION) > 3.8) && empty($conf->global->SEARCHEVERYWHERE_SEARCH_PREVIEW)) {
```

**Comment by cfoellmann** [comment]
@atm-john 🤦 
force pushed

**Comment by cfoellmann** [comment]
can this be merged?

**Comment by atm-john** [comment]
main is fixed

**Pull request closed without merging**